From f04a5c9d6a733f44d8f88dcea4ec340ec2d1d2c4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sat, 10 Dec 2016 13:01:22 -0500
Subject: [PATCH 24/59] build-sys: define arm as secondary architecture for
 arm64

Completely unstested. Fixes #4862.

(cherry picked from commit b7cac52779d52eff6ed03103922ffe1208a7baa9)
---
 src/basic/architecture.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/basic/architecture.h b/src/basic/architecture.h
index 5a77c31..b329df2 100644
--- a/src/basic/architecture.h
+++ b/src/basic/architecture.h
@@ -150,6 +150,7 @@ int uname_architecture(void);
 #  else
 #    define native_architecture() ARCHITECTURE_ARM64
 #    define LIB_ARCH_TUPLE "aarch64-linux-gnu"
+#    define SECONDARY_ARCHITECTURE ARCHITECTURE_ARM
 #  endif
 #elif defined(__arm__)
 #  if __BYTE_ORDER == __BIG_ENDIAN
-- 
2.8.3

